SConstruct

```diff
@@ -221,12 +248,15 @@ if platform == 'darwin':
 # Sometimes it's easier just to copy the dependencies into the repo dir
 # We try to auto-detect this.
 if path.exists('deps/lib'):
-	env.Append(LIBPATH=['deps/lib'])
+	env.Append(LIBPATH=[os.getcwd() + '/deps/lib'])
 	if platform == 'darwin':
-		env.Append(FRAMEWORKPATH=['deps/lib'])
+		env.Append(FRAMEWORKPATH=[os.getcwd() + '/deps/lib'])
+
+if path.exists('deps/lib64'):
+	env.Append(LIBPATH=[os.getcwd() + '/deps/lib64'])
 
 if path.exists('deps/include'):
-	env.Append(CPPPATH=['deps/include'])
+	env.Append(CPPPATH=[os.getcwd() + '/deps/include'])
```

Comment:
I'm not really a fan of concatenating `os.getcwd()` like this. I'd prefer to use a dedicated path join function (either `os.path` or `pathlib`).